src/main/scala/viper/gobra/ast/frontend/Ast.scala

```diff
@@ -691,6 +691,11 @@ case class PFunctionSpec(
                       isPure: Boolean = false,
                       ) extends PSpecification
 
+case class PBodyParameterInfo(
+                           /** stores indices of parameters that are used as addressable variables in the code body */
```

Comment:
`indices` seems a bit inappropriate as you're storing IdnUses

Reply:
you are right. Before I used a set on integers.